### PR TITLE
Adds 3.9.1 release notes

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

medic/cht-core#6593

# Code review items

- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: release-notes/docs/3.9.1.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+There are no required changes to the [supported software matrix](https://github.com/medic/medic-docs/blob/master/installation/supported-software.md) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+### Bug fixes
+
+- [cht-core#6562](https://github.com/medic/cht-core/issues/6562): Muso production users have multiple task documents in non-terminal state with the same emission id
```
Comment (garethbowen) [inline release-notes/docs/3.9.1.md:13]: Rename this issue (and then update this doc) with a more generic description so it's clear that it affects everyone not just Muso.

**Review by garethbowen — CHANGES_REQUESTED** [review]
One small change requested.

File: release-notes/docs/3.9.1.md (hunk 1 above)
Comment (dianabarsan) [reply release-notes/docs/3.9.1.md]: Done!

File: release-notes/docs/3.9.1.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+There are no required changes to the [supported software matrix](https://github.com/medic/medic-docs/blob/master/installation/supported-software.md) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+### Bug fixes
+
+- [cht-core#6562](https://github.com/medic/cht-core/issues/6562): Switching between tabs while tasks are calculated can result in having multiple task documents with the same emission id
```
Comment (garethbowen) [inline release-notes/docs/3.9.1.md:13]: I'm not sure about the grammar here. I've updated the issue title to...

```suggestion
- [cht-core#6562](https://github.com/medic/cht-core/issues/6562): Switching between tabs while tasks are being calculated can result in having multiple task documents with the same emission id
```

**Review by garethbowen — APPROVED** [review]
Approved pending one optional suggestion.

File: release-notes/docs/3.9.1.md (hunk 2 above)
Comment (dianabarsan) [reply release-notes/docs/3.9.1.md]: Thanks.

**Pull request merged**